.github/workflows/dotnet-deploy.yml

```diff
@@ -20,8 +20,8 @@
 name: Build and deploy ASP.Net Core app to an Azure Web App
 
 env:
-  AZURE_WEBAPP_NAME: interview-app                # set this to the name of your Azure Web App
-  AZURE_WEBAPP_PACKAGE_PATH: './src/WebApi'       # set this to the path to your web app project, defaults to the repository root
+  AZURE_WEBAPP_NAME: api-interview-dev            # set this to the name of your Azure Web App

```

Comment:
If we're using this file for deploying the API to development environment, then I suppose we'll use a separate file for production deployment. Might consider renaming the .yml file 😊

Reply:
So far this is not an issue and dotnet-deploy is fine. We are far from production and there is an option to use the same workflow with different params. We will delay the decision on how to approach that.